web-external/src/components/dialog/register/footer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Footer extends React.Component {
+  render () {
+    let {
+      onClose,
+      onSubmit
+    } = this.props;
+
+    return (
+      <div className='modal-footer'>
+        <a className='btn btn-default'
+           data-dismiss='modal'
+           onClick={ onClose }>
+          Close
+        </a>
+        <button className='btn btn-primary'
+                id='g-register-button'
+                type='submit'
+                onClick={ onSubmit }>
+          <i className='icon-plus-circled'>Register</i>
```

Comment:
This makes `Register` appear in italics. Follow the example of `login/footer.jsx` and put a slash before the `i` tag's closing bracket, and remove the `</i>`.